### PR TITLE
changed to use npm modules + browserify

### PR DESCRIPTION
**Pull request by edewit**
fixes issue [RHMAP-7023](https://issues.jboss.org/browse/RHMAP-7023)

### CODE REVIEW

**Comment by secondsun** [comment]
@edewit Is building the app in studio sufficient to verify?

File: Gruntfile.js [hunk 1]
```diff
@@ -78,7 +78,11 @@ module.exports = function (grunt) {
     // Empties folders to start fresh
     clean: {
       server: '.tmp'
-    }
+    },
+
+    browserify: {
+      'www/main.js': ['www/js/libs/**/*.js', 'www/js/app/views/viewbase.view.js', 'www/js/app/**/*.js']
+    },
```
Comment (jcesarmobile) [inline Gruntfile.js]: Should not end with `,`

**Comment by jcesarmobile** [comment]
Just one minor comment, but other than that LGTM

As we talked on slack,  we have to add 'npm install' task to digger to install all the new dependencies.

**Comment by jcesarmobile** [comment]
Ah, just forgot, `www/js/app/templates.js` was deleted, not sure if that was on purpose because it was just not used.

**Comment by edewit** [comment]
@jcesarmobile it was not used

**Comment by jcesarmobile** [comment]
LGTM

**Pull request merged**